src/mongocxx/collection.hpp

```diff
@@ -466,6 +466,20 @@ class MONGOCXX_API collection {
     stdx::string_view name() const noexcept;
 
     ///
+    /// Rename this collection.
+    ///
+    /// @param new_name The new name to assign to the collection.
+    /// @param drop_target_before_rename Whether to overwrite any
+    ///   existing collections called new_name. The default is false.
+    ///
+    /// @throws exception::operation if the operation fails.
+    ///
+    /// @see https://docs.mongodb.org/manual/reference/command/renameCollection/
+    ///
+    void rename(stdx::string_view new_name, bool drop_target_before_rename = false);
```

Comment:
Yes. Any method that needs an "and" to describe it is not a method. It frustrates error handling as well: which part failed? However, we do have a spec for the CRUD API, so if it is mandated we should try to adhere to the spec. If the specification does mandate that this method take an argument to describe the drop target behavior, I'd at least prefer that we make it an enum so you can understand it when seen in client code without needing to look up the driver headers.